src/shared/components/mutationTable/column/AnnotationColumnFormatter.tsx

```diff
@@ -57,31 +57,10 @@ export default class AnnotationColumnFormatter
         };
     }
 
-    /**
-     * Derives status from data content.
-     * Default (empty) data means loading. Null data means error.
-     */
-    public static getOncoKbStatus(oncoKbData?: IOncoKbData): "complete"|"loading"|"error"
```

Comment:
We don't need to derive status from data anymore, since we are passing the complete `MobXPromise` object to the components